tokio/src/task/join_set.rs

```diff
@@ -251,6 +261,9 @@ impl<T: 'static> JoinSet<T> {
     /// provided runtime and store it in this `JoinSet`, returning an
     /// [`AbortHandle`] that can be used to remotely cancel the task.
     ///
+    /// Note that this task can only be aborted before it starts running.
+    /// This is because they are not *async*, see [task cancellation].
+    ///
     /// [`AbortHandle`]: crate::task::AbortHandle
     #[track_caller]
```

Comment:
```suggestion
    /// This is because they are not *async*, see [task cancellation].
    ///
    /// [`AbortHandle`]: crate::task::AbortHandle
    /// [task cancellation]: crate::task#cancellation
    #[track_caller]
```